crates/jxl-frame/src/lib.rs

```diff
@@ -120,6 +120,14 @@ impl<'a> Bundle<&'a ImageHeader> for Frame<'a> {
             }
         }
 
+        if header.have_crop
+            && ((header.width == 0) | (header.height == 0))
```

Comment:
```suggestion
        if header.have_crop
            && (header.width == 0 || header.height == 0)
```

Also, you can just remove `header.have_crop` there, because `width` and `height` will reflect those in the image header if `!have_crop`.